File: backend/src/main/java/org/example/backend/ResponseModels/TotalStepsOfUsers.java
package org.example.backend.ResponseModels;

import lombok.Data;

@Data
public class TotalStepsOfUsers {

    private long userId;
    private String name;
    private long totalSteps;

    public TotalStepsOfUsers(long userId, String name, long totalSteps) {
        this.userId = userId;
        this.name = name;
        this.totalSteps = totalSteps;
    }
}

File: frontend/src/models/TotalStepsOfUsers.ts

export class TotalStepsOfUsers {
    userId: number
    name: string;
    totalSteps: number;

    constructor(userId: number, name: string, totalSteps: number) {
        this.userId = userId;
        this.name = name;
        this.totalSteps = totalSteps;
    }
}